### DISCUSSION TITLE
Including ansys-dpf-core in an executable

### DISCUSSION THREAD

**Discussion by Indrajeetrt** [ansys/pydpf-core#1826] [Q&A]
Hello there, Good day.. I have a script which uses pydpf-core and for user interaction i have used tkinter gui, but when create exe file using autp-py-to-exe library i am getting the bellow error
"""
ansys\dpf\core\data_sources.py:395: UserWarning: Traceback (most recent call last):
  File "ansys\dpf\core\data_sources.py", line 393, in __del__
AttributeError: 'DataSources' object has no attribute '_deleter_func'
"""
Can someone help in debugging this ?
Thank you in advance

**Comment by sandeep-tubs** [comment]
Hey, I have the same issue. I'm using python 3.11 & trying to create exe. Can someone help please?

**Comment by PProfizi** [comment]
Hello @Indrajeetrt, @sandeep-tubs,

I understand you both included `ansys-dpf-core` as a component for an application for which you generated an `exe` file using `auto-py-to-exe`.

From the error you get I am wondering if, when running your application, `ansys-dpf-core` finds a DPF server installation to run.
Are you providing access to an ANSYS installation or to a DPF Server to `ansys-dpf-core`?
To learn more about how `ansys-dpf-core` finds a DPF Server installation, please check [this documentation section](https://dpf.docs.pyansys.com/version/stable/getting_started/dpf_server.html#manage-multiple-dpf-server-installations).
Maybe also check [this issue](https://github.com/ansys/pydpf-core/issues/1382) where we resolved a problem of missing dependencies.
